<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" name="Failed suite [KTCTC]">
  <test thread-count="5" name="Regression(failed)" parallel="classes">
    <classes>
      <class name="invocation.Invocation">
        <methods>
          <include name="verifyUserCanLoginToApplicationWithValidCredentials"/>
        </methods>
      </class> <!-- invocation.Invocation -->
      <class name="assertions.SoftAssertionInTestNG">
        <methods>
          <include name="verifyUserCanLoginToApplicationWithValidCredentials"/>
          <include name="sampleAssertionTestCase"/>
          <include name="verifyUserCanNOtLoginToApplicationWithInvalidCredentials"/>
        </methods>
      </class> <!-- assertions.SoftAssertionInTestNG -->
      <class name="dependent.Dependent">
        <methods>
          <include name="verifyUserCanLoginToApplicationWithValidCredentials"/>
          <include name="verifyUserCanSeeSentEmailInSentBox"/>
          <include name="verifyUserCanSendEmail"/>
        </methods>
      </class> <!-- dependent.Dependent -->
      <class name="assertions.AssertionInTestNG">
        <methods>
          <include name="verifyUserCanNOtLoginToApplicationWithInvalidCredentials"/>
          <include name="sampleAssertionTestCase"/>
          <include name="verifyUserCanLoginToApplicationWithValidCredentials"/>
        </methods>
      </class> <!-- assertions.AssertionInTestNG -->
    </classes>
  </test> <!-- Regression(failed) -->
</suite> <!-- Failed suite [KTCTC] -->
